src/prisma/schema.prisma

```diff
@@ -2283,12 +2290,19 @@ model Project {
   description    String?
   twitter        String?
   website        String?
+  websites       String[]        @default([])
+  farcasterIds   String[]        @default([])
```

Comment:
Question; what does farcasterIds refer to here? Is this the list of team members?

Reply:
@Devorein this is not necessarily ids even, it should just be "farcaster: []" might be fine (thats how we got it from agora) . most of the time the string is a URL, sometimes it is "@username", sometimes just "TBD"

Reply:
Good point. It shouldn't contain the chunk ids as thats misleading. Probably `farcasterValues` is better to indicate a plurality of values.